client/ayon_usd/standalone/Usd/Pinning/pinning_file_helper.py

```diff
@@ -0,0 +1,235 @@
+import json
+import os
+import re
+from typing import Dict, List
+from pxr import UsdShade, Ar, Sdf
+from urllib.parse import urlparse
+
+
+def is_uri(path):
+    parsed = urlparse(path)
+    return parsed.scheme
```

Comment:
Does `parsed.scheme` return a bool? The function name makes me think this function will return a bool.

```suggestion
def is_uri(path: str) -> bool:
    parsed = urlparse(path)
    return parsed.scheme
```

According to [the docs](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse) it returns a `str` which is confusing maybe for this function name?


```suggestion
def is_uri(path: str) -> bool:
    parsed = urlparse(path)
    return bool(parsed.scheme)
```